packages/account-v2/src/modules/ManualForm/manualForm.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { Form, Formik } from 'formik';
+import { Button } from '@deriv/ui';
+import { MANUAL_FORM_INITIAL_VALUES, TManualDocumentTypes } from '../../constants/manualFormConstants';
+import { getManualFormValidationSchema } from '../../utils/manualFormUtils';
+import { ManualFormDocumentUpload } from './manualFormDocumentUpload';
+import { ManualFormFooter } from './manualFormFooter';
+import { ManualFormInputs } from './manualFormInputs';
+
+type TManualFormProps = {
+    selectedDocument: TManualDocumentTypes;
+};
+
+const Divider = () => <div className='w-full border-t-solid-grey-5 border-solid border-t-100' />;
```

Comment:
Should be in `deriv-com/ui`

Reply:
i have removed this component from the PR, and used margin color instead.